app/views/layouts/_shim.html.erb

```diff
@@ -0,0 +1,4 @@
+<!--[if lt IE 9]>
+  <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/r29/html5.min.js">
```

Comment:
please remove this file when not used yet

Reply:
I have to add this file. It will become error if I delete it.